packages/graphback-codegen-offix/src/OffixPlugin.ts

```diff
@@ -80,14 +83,15 @@ export class OffixPlugin extends GraphbackPlugin {
                 schemaComposer.Query.addFieldArgs(diffQuery, {
                     lastSync: 'String!'
                 })
-                // TODO generate delta resolvers
             }
         })
 
-        return schemaComposer.buildSchema()
+        return buildSchema(schemaComposer.toSDL())
```

Comment:
`schemaComposer.toSDL()` returns defaults scalars so this hack allows us to get the right schema.